activiti-spring-boot-starter/src/test/resources/processes/initial-vars-extensions.json

```diff
@@ -0,0 +1,34 @@
+{
+  "id":"initialVarsProcess",
```

Comment:
@balsarori @ryandawsonuk where is "id" and "name" stored in the model created by this PR? We cannot read half of the information.

Reply:
What I would suggest is to have a model for the entire file.

Reply:
``ProcessExtensionModel`` represents this file. Seems that the name is missing from that class but we don't use it. It could be added. I can check what the modeler produces and make sure that all properties a represented.

Reply:
@balsarori then all the information needs to be present and validated.. so is ID covered already?

Reply:
ID at the top level is [in ProcessExtensionModel](https://github.com/Activiti/Activiti/blob/ca05e8b8b4825e3f70dc7ecc88e1a5a741bebb9e/activiti-spring-process-extensions/src/main/java/org/activiti/spring/process/model/ProcessExtensionModel.java#L18) then below that [in VariableDefintion](https://github.com/Activiti/Activiti/pull/2195/files#diff-a86fb2b9c77a756fe46931bb51c01a31R18)